### ISSUE TITLE
3.3.0-m3 addons randomly don't work after a restart

### ISSUE THREAD

**Issue by sim0nx** [openhab/openhab-core#2938]
I have recently upgraded to 3.3.0-m3 in order for the tesla addon to work correctly (new/changed auth method).
This introduced a weird issue for me that at restart, one or more addons don't work at all, i.e. they are shown as being online but items never get updated.
Trying to disable/enable such addons takes forever and does not make them work (this is logged as such).
SSH'ing into openhab and doing it from there also does not work or sometimes does, but re-enabling them keeps them in uninitialized state.
Restarting openhab again randomly makes some work again while others fail, meaning those that work and those that don't isn't consistent over restarts.

This feels like a race condition somewhere, though I have no clue where to look really.

Affected addons in my case are:
- tesla
- hue
- openweathermap
- nuki

**Comment by J-N-K** [comment]
What exactly is in the log?

**Comment by sim0nx** [comment]
Currently after a fresh restart, all things are displayed as being *online*.
The tesla addon does not post any updates though.

Disabling the tesla addon (bridge) gives:
```
<bridge> updated: UNINITIALIZED
<bridge> changed from ONLINE to UNINITIALIZED
Disposing handler for thing .... takes more than 5000ms.
<bridge> updated: UNINITIALIZED
<bridge> changed from ONLINE to UNINITIALIZED
<bridge> updated: UNINITIALIZED (DISABLED)
<bridge> changed from UNINITIALIZED to UNINITIALIZED (DISABLED)
<car instance> updated: UNINITIALIZED (BRIDGE_UNINITIALIZED)
<car instance> changed from UNINITIALIZED to UNINITIALIZED (BRIDGE_UNINITIALIZED)
```

Re-enabling the bridge:
```
<bridge> updated: INITIALIZING
<bridge> changed from UNINITIALIZED (DISABLED) to INITIALIZING
<bridge> updated: UNKNOWN
<bridge> changed from INITIALIZING to UNKNOWN
<car instance> updated: INITIALIZING 
<car instance> changed from UNINITIALIZED (BRIDGE_UNINITIALIZED) to INITIALIZING
<bridge> updated: ONLINE
<bridge> changed from UNKNOWN to ONLINE
<bridge> updated: ONLINE
Initializing handler for thing ...... takes more than 5000ms.
```

The *<car instance>* stays in *INITIALIZING* until I restart openhab again.

**Comment by sim0nx** [comment]
My issue seems to be related to
https://github.com/openhab/openhab-addons/pull/12603

Fixed in 3.3.0M5
